poly-commitment/src/srs.rs

```diff
@@ -219,6 +229,39 @@ where
         self.lagrange_bases.insert(n, chunked_commitments);
     }
 
+    /// This function creates a trusted-setup SRS instance for circuits with number of rows up to `depth`.
```

Comment:
How is the waste `x` being dealt here? Meaning, there is no updateable approach here right, but truly trusted, right?